examples/gno.land/r/demo/users/users.gno

```diff
@@ -235,6 +235,20 @@ func GetUserByAddressOrName(input users.AddressOrName) *users.User {
 	return GetUserByAddress(std.Address(input))
 }
 
+// Get a list of user names starting from the given prefix. Limit the
+// number of results to maxResults. (This can be used for a name search tool.)
+func ListUsersByPrefix(prefix string, maxResults int) []string {
```

Comment:
as suggested by @iuricmp, we could have a `p/demo/avlhelpers` package with such generic helpers.

Reply:
The latest commit adds p/demo/avlhelpers.